From 84b28bcebd2bdf6fb908bb3786e73312f0f75389 Mon Sep 17 00:00:00 2001
From: Andi Kleen <andi@firstfloor.org>
Date: Sat, 3 Apr 2021 09:42:29 -0700
Subject: [PATCH 35/39] softirq/lto: Make irq_exit_rcu visible

irq_exit_rcu is called from assembler, thus needs to be __visible with
gcc LTO.

Signed-off-by: Andi Kleen <andi@firstfloor.org>
---
 kernel/softirq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/softirq.c b/kernel/softirq.c
index 9908ec4a9bfe..462c8f6b8709 100644
--- a/kernel/softirq.c
+++ b/kernel/softirq.c
@@ -429,7 +429,7 @@ static inline void __irq_exit_rcu(void)
  *
  * Also processes softirqs if needed and possible.
  */
-void irq_exit_rcu(void)
+__visible void irq_exit_rcu(void)
 {
 	__irq_exit_rcu();
 	 /* must be last! */
-- 
2.30.2

